readme.md
<img alt="Banner" style="border-width:0" src="http://i273.photobucket.com/albums/jj219/johnsmm/banner.jpg" />
## Created by Benny and OFPS Team
<br>
Main repo, for Stratis only atm
## Check the new [Repo Workflow](http://gitlab.ofps.net/ofps/BECTI-Benny-Warfare-OFPS.Stratis/wikis/repo-workflow)


changelog.txt
Upcoming release
Fixed: Hill climb, push, and taxi reverse not applying to town occupation vehicles
Removed: Opfor gaz66 medical vehicle
Added: 10 AT disposable launchers to each starting vehicles
Fixed: floating Camp in takistan

1.06++
Fixed: Rearm on some aircraft giving incorrect loadouts
Tweaked: rearming an aicraft doesn't remove all of this ammo during rearm. 
Fixed: tool kit repair not always fixing tires on a vehicle
Tweaked: tool kit repair no longer repairs reactive armor on rhs tanks
Tweaked: Tool Kit repair repairs damaged components to a maxium of 75% operational strength
Tweaked: Rollup mbt prices +5%/+10%/+15%
Replace: obyekt-681with bmd-4m (a) at opfor hf3
Tweaked: Cyrus 9.3mm to opfor instead of navid
Added: Remaining smoke grenades
Tweaked: Correctly tier titan at(g3) and aa(g4)
Tweaked: Unified blufor/opfor titan ammo cost (ap800/aa1500/at2500)
Tweaked: Balance blufor/opfor lack of thermals

1.06+
Fixed: Depot available when camps are lost
Replaced: Opfor mbt's with t72d/t80bkv/t80ue-1
Tweaked: Reduced all AF prices (AF4-5%/AF3-10%/AF2-15%/AF1-20%/AF0-25%)

1.06
Tweaked: Moved l39 from air factory 1 to air factory 2
Tweaked: Moved BRDM ATGM from light factory 0 to light factory 2
Tweaked: Moved kajman from air factory 3 to air factory 4
Tweaked: Changed price of orca armed from 18500 to 22500
Tweaked: Changed price of kajman from 45000 to 50000
Tweaked: Changed price of apache (empty) from 15000 to 17500
Tweaked: Changed price of apache rocket pods from 20000 to 22500
Tweaked: Changed price of apache AT from 40000 to 50000
Tweaked: Changed price of ah-1z cobra (empty) from 15000 to 16000
Tweaked: Changed price of ah-1z cobra rocket pods from 20000 to 22500
Tweaked: Changed price of ah-1z cobra (AT) from 40000 to 50000
Tweaked: Changed price of AH6 At (little bird AT) from 14500 to 22500
Tweaked: Changed price of AH-99 Blackfoot from 65000 to 55000
Tweaked: Changed price of Mi-24P Hind from 20000 to 25000
Tweaked: Changed price of KA-52 from 60000 to 50000
Tweaked: Player and AI bodies now despawn immediately after death
Tweaked: Ai bodies now despawn after disband
Tweaked: SAM - Engagement minimum alt 50m and rockets been modified to create less damage.
Tweaked: SAM - Both sides share same Sam now, with new textures
Tweaked: SAM - Smoke from sam rockets been replaced with vanilla smoke to create less fps drop.
Tweaked: SAM - Will fire every 15 sec to 4k range. 
Tweaked: Base health increased damage from AP shells, and decreased from HE canons and arty.
Tweaked: Gear on player reconnect wasn’t saving some times.
Tweaked: Cash flow decreased and supply rate increased by 15%
Tweaked: Adjusted air lift for all mid-size choppers to lift tanks.
Tweaked: Restructured base defense build menu. 
Tweaked: Fuel consumption decrease to about 15-20%
Tweaked: Base under attack hints will display more frequent. 
Tweaked: Removed town circles from friendly towns to declutter the map a bit.
Fixed: Peace time marker will not show.
Fixed: Adjusted defense factions for Cram and all Sam units.
Fixed: No SAM warning for players without vehicles or in parachute
Fixed: Slammer up not purchasable on non-snow maps
Fixed: Slammer not purchasable on snow maps
Fixed: Titan AT launcher not accessible in gear 3 for blufor
Fixed: Ruins from base buildings (such as base defense domes) now despawn quicker. 
Fixed: Camps relocated inside town circle for some towns.
Fixed: Town circle will disappear in some towns.
Fixed: When player teamkills chat will display players name
Fixed: When using toolkit to repair you could skip it to get instant repair.
Fixed: Defense object collision.
Added: Indicator hint for player shooting an enemy structure that contains the current health of the structure after shot
Added: Hint for players if a friendly structure is attacked. displays health after damage
Added: Friendly team notifications for when HQ is mobilized and deployed
Added: Barracks upgrade level will increase player AI slots and skill
Added: Camp award bounty
Added: Depot purchase menu disabled if camp is lost to other side.
Added: Low grass mod to core pack.
Added: RHS Trucks to be able to load cargo such as ammo creates 
Changed: Removed C-RAM2 (Missiles) as it is inferior to the normal C-RAM
Removed: Some fortifications from defense truck menu.
Removed: Items from defense truck and repair truck old action menus.
Removed: Actions menus from halo drop.
Tweaked: Halo jump cooldown now changed from 1 minute to 5 minutes.

1.05
Added: Spawn on vehicle disabled when enemy nearby 50M
Added: Tank Low Gear / Hill climb
Added: Tank Stealth mode
Added: Parameter to set all camps required to capture town
Added: All camps must be captured to capture town
Added: Base area enemy reporting satellite upgrade

rsc/parameters.hpp
class Params {
	class CTI_FACTION_WEST {
		title = "FACTION: West Forces";
		values[] = {0,1,2};
		texts[] = {"NATO Arid","NATO Tropic","NATO Winter"};
		default = 0;
	};
	class CTI_FACTION_EAST {
		title = "FACTION: East Forces";
		values[] = {0,1,2};
		texts[] = {"CSAT Arid","CSAT Tropic","CSAT Winter"};
		default = 0;
	};
	class CTI_FACTION_DEFAULT_VEHICLES {
		title = "FACTION: Starting vehicles and mhq";
		values[] = {0,1,2,3};
		texts[] = {"Vanilla","Apex","CUP","RHS"};
		default = 3;
	};
	class CTI_FACTION_DEFAULT_GEAR {
		title = "FACTION: Starting gear";
		values[] = {0,1,2,3};
		texts[] = {"Vanilla","Apex","CUP","RHS"};
		default = 3;
	};
	class CTI_FACTION_DEFAULT_TROOPS {
		title = "FACTION: Default infantry";
		values[] = {0,1,2,3};
		texts[] = {"Vanilla","Apex","CUP","RHS"};
		default = 3;
	};
	class SEPARATOR0 {
		title = "========================== INCOME ============================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_BASE_AREA_MAX {
		title = "BASE: Areas";
		values[] = {1,2,3,4,5};
		texts[] = {"1","2","3","4","5"};
		default = 2;
	};
	class CTI_BASE_HQ_REPAIR {
		title = "BASE: HQ Repairable";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 1;
	};
	class CTI_BASE_FOB_MAX {
		title = "BASE: FOB Limit";
		values[] = {0,1,2,3,4,5,6,7,8,9,10};
		texts[] = {"Disabled","1","2","3","4","5","6","7","8","9","10"};
		default = 2;
	};
	class CTI_BASE_LARGE_FOB_MAX {
		title = "BASE: Large FOB Limit";
		values[] = {0,1,2,3,4,5,6,7,8,9,10};
		texts[] = {"Disabled","1","2","3","4","5","6","7","8","9","10"};
		default = 2;
	};
	class CTI_BASE_STARTUP_PLACEMENT {
		title = "BASE: Startup Placement";
		values[] = {2000,3000,4000,5000,6000,7000,8000,9000,10000,12000,15000,20000};
		texts[] = {"2 KM","3 KM","4 KM","5 KM","6 KM","7 KM","8 KM","9 KM","10 KM","12 KM","15 KM","20 KM"};
		default = 9000;
	};
	class SEPARATOR1 {
		title = "========================== INCOME ============================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_ECONOMY_INCOME_CYCLE {
		title = "INCOME: Delay";
		values[] = {30,35,40,45,50,55,60,65,70,75,80};
		texts[] = {"30 Seconds","35 Seconds","40 Seconds","45 Seconds","50 Seconds","55 Seconds","01:00 Minutes","01:05 Minutes","01:10 Minutes","01:15 Minutes"};
		default = 30;
	};
	class CTI_ECONOMY_STARTUP_FUNDS_EAST_COMMANDER {
		title = "INCOME: Starting Funds (East Commander)";
		values[] = {9000,15000,20000,25000,30000,35000,40000,45000,50000,60000,100000000};
		texts[] = {"$9000","$15000","$20000","$25000","$30000","$35000","$40000","$45000","$50000","$60000","$100000000"};
		default = 30000;
	};
	class CTI_ECONOMY_STARTUP_FUNDS_EAST {
		title = "INCOME: Starting Funds (East Players)";
		values[] = {900,1500,2400,3200,6000,8000,10000,12500,15000,20000,100000000};
		texts[] = {"$900","$1500","$2400","$3200","$6000","$8000","$10000","$12500","$15000","$20000","$100000000"};
		default = 12500;
	};
	class CTI_ECONOMY_STARTUP_FUNDS_WEST_COMMANDER {
		title = "INCOME: Starting Funds (West Commander)";
		values[] = {9000,15000,20000,25000,30000,35000,40000,45000,50000,60000,100000000};
		texts[] = {"$9000","$15000","$20000","$25000","$30000","$35000","$40000","$45000","$50000","$60000","$100000000"};
		default = 30000;
	};
	class CTI_ECONOMY_STARTUP_FUNDS_WEST {
		title = "INCOME: Starting Funds (West Players)";
		values[] = {900,1500,2400,3200,6000,8000,10000,12500,15000,20000,100000000};
		texts[] = {"$900","$1500","$2400","$3200","$6000","$8000","$10000","$12500","$15000","$20000","$100000000"};
		default = 12500;
	};
	class CTI_ECONOMY_TOWNS_OCCUPATION {
		title = "INCOME: Towns Occupation";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 1;
	};
	class CTI_ECONOMY_STARTUP_SUPPLY_EAST {
		title = "SUPPLY: Starting Supply (East Team)";
		values[] = {9000,15000,20000,25000,30000,35000,40000,45000,50000,60000,100000000};
		texts[] = {"$9000","$15000","$20000","$25000","$30000","$35000","$40000","$45000","$50000","$60000","$100000000"};
		default = 20000;
	};
	class CTI_ECONOMY_STARTUP_SUPPLY_WEST {
		title = "SUPPLY: Starting Supply (West Team)";
		values[] = {9000,15000,20000,25000,30000,35000,40000,45000,50000,60000,100000000};
		texts[] = {"$9000","$15000","$20000","$25000","$30000","$35000","$40000","$45000","$50000","$60000","$100000000"};
		default = 20000;
	};
	class SEPARATOR2 {
		title = "========================== TOWNS ============================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_TOWNS_OCCUPATION {
		title = "TOWNS: Occupation";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 1;
	};
	class CTI_TOWNS_OCCUPATION_SKILL {
		title = "TOWNS: Occupation Skill level";
		values[] = {20,30,40,50,60,70,80,90,100};
		texts[] = {"20","30","40","50","60","70","80","90","100"};
		default = 80;
	};
	class CTI_TOWNS_OCCUPATION_LEVEL_RESISTANCE {
		title = "TOWNS: Resistance Occupation Squad Count";
		values[] = {5,10,15,20,25,30,35,40};
		texts[] = {"Amateur","Novice","Average","Skilled","Professional","Specialist","Expert","Chuck Norris"};
		default = 20;
	};
	class CTI_TOWNS_OCCUPATION_LEVEL {
		title = "TOWNS: Side Occupation Squad Count";
		values[] = {5,10,15,20,25,30,35,40};
		texts[] = {"Amateur","Novice","Average","Skilled","Professional","Specialist","Expert","Chuck Norris"};
		default = 20;
	};
	class CTI_TOWNS_OCCUPATION_RESISTANCE {
		title = "TOWNS: Resistance Occupation Forces";
		values[] = {0,1,2,3,4,5,6,7,8,9};
		texts[] = {"Vanilla - AAF","Vanilla - FIA","CUP - ION PMC","CUP - NAPA Chernarus","CUP - Royal Army Corp Of Sahrani","CUP - Takistani Military","Mixed","Syndikat Paramilitary","AAF/Swedish Winter","RHS GREF"};
		default = 9;
	};
	class CTI_TOWNS_OCCUPATION_WEST {
		title = "TOWNS: Blufor Occupation Forces";
		values[] = {0,1,2,3,4,5};
		texts[] = {"Vanilla","CUP - US Army","Mixed","Pacific Special Forces","Winter","RHS"};
		default = 5;
	};
	class CTI_TOWNS_OCCUPATION_EAST {
		title = "TOWNS: Opfor Occupation Forces";
		values[] = {0,1,2,3,4,5};
		texts[] = {"Vanilla","CUP - Russians","Mixed","Pacific Special Forces","Winter","RHS"};
		default = 5;
	};
	class CTI_TOWNS_PEACE {
		title = "TOWNS: Peace";
		values[] = {0,60,120,180,300,600};
		texts[] = {"Disabled","1 Minute","2 Minutes","3 Minutes","5 Minutes","10 Minutes"};
		default = 300;
	};
	class CTI_TOWNS_OCCUPATION_LIMIT_AI {
		title = "TOWNS: Occupation Soft max AI Limit";
		values[] = {50,100,150,200,250,300,350,400};
		texts[] = {"50","100","150","200","250","300","350","400"};
		default = 100;
	};
	class CTI_TOWNS_OCCUPATION_LIMIT_AI_QUEUE_RATIO {
		title = "TOWNS: Occupation Squad Queue Ratio";
		values[] = {0,10,20,30,40,50,60,70,80,90,100};
		texts[] = {"0","10","20","30","40","50","60","70","80","90","100"};
		default = 20;
	};
	class CTI_TOWNS_RESISTANCE_LIMIT_AI {
		title = "TOWNS: Resistance Soft max AI Limit";
		values[] = {50,100,150,200,250,300,350,400};
		texts[] = {"50","100","150","200","250","300","350","400"};
		default = 150;
	};
	class CTI_TOWNS_RESISTANCE_LIMIT_AI_QUEUE_RATIO {
		title = "TOWNS: Resistance Squad Queue Ratio";
		values[] = {0,10,20,30,40,50,60,70,80,90,100};
		texts[] = {"0","10","20","30","40","50","60","70","80","90","100"};
		default = 20;
	};
	class CTI_TOWNS_CAPTURE_MODE {
		title = "TOWNS: CAPTURE MODE";
		values[] = {0,1,2};
		texts[] = {"Normal","Camp Cap Boost","All Camps Required (also boosts)"};
		default = 1;
	};
	class CTI_TOWNS_TERRITORIAL {
		title = "TOWNS: Territorial Mode";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 0;
	};
	class SEPARATOR3 {
		title = "========================== RESPAWN ============================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_RESPAWN_AI {
		title = "RESPAWN: AI Members";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 0;
	};
	class CTI_RESPAWN_CAMPS {
		title = "RESPAWN: Camps";
		values[] = {0,1,2};
		texts[] = {"Disabled","Town Classic","Nearby Camps"};
		default = 2;
	};
	class CTI_RESPAWN_CAMPS_CONDITION {
		title = "RESPAWN: Camps Condition";
		values[] = {0,1,2};
		texts[] = {"No limits","Priced","Limited"};
		default = 0;
	};
	class CTI_RESPAWN_FOB_RANGE {
		title = "RESPAWN: FOB Range";
		values[] = {500,750,1000,1250,1500,1750,2000};
		texts[] = {"0.50 KM","0.75 KM","1 KM","1.25 KM","1.5 KM","1.75 KM","2 KM"};
		default = 1500;
	};
	class CTI_RESPAWN_LARGE_FOB_RANGE {
		title = "RESPAWN: Large FOB Range";
		values[] = {500,750,1000,1250,1500,1750,2000,2250,2500,2750,3000};
		texts[] = {"0.50 KM","0.75 KM","1 KM","1.25 KM","1.5 KM","1.75 KM","2 KM","2.25 KM","2.5 KM","2.75 KM","3 KM"};
		default = 2500;
	};
	class CTI_RESPAWN_MOBILE {
		title = "RESPAWN: Mobile";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 1;
	};
	class CTI_RESPAWN_TIMER {
		title = "RESPAWN: Delay";
		values[] = {15,20,25,30,35,40,45,50,55,60};
		texts[] = {"15 Seconds","20 Seconds","25 Seconds","30 Seconds","35 Seconds","40 Seconds","45 Seconds","50 Seconds","55 Seconds","60 Seconds"};
		default = 15;
	};
	class SEPARATOR4 {
		title = "============================ AI ==============================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_AI_TEAMS_ENABLED {
		title = "AI: Controlled Teams";
		values[] = {0,1};
		texts[] = {"Disabled", "Enabled"};
		default = 1;
	};
	class CTI_PLAYERS_GROUPSIZE {
		title = "AI: Player Group Size";
		values[] = {0,1,2,3,4,5,8,10,12,14,16};
		texts[] = {"Rank Based","1","2","3","4","5","8","10","12","14","16"};
		default = 0;
	};
	class SEPARATOR5 {
		title = "=========================== UNITS ============================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_VEHICLES_AIR_FFAR {
		title = "UNITS: Aircraft FFAR";
		values[] = {0,1,2};
		texts[] = {"Disabled","Enabled with Upgrade","Enabled"};
		default = 2;
	};
	class CTI_VEHICLES_AIR_DAR {
		title = "UNITS: Aircraft DAR";
		values[] = {0,1,2};
		texts[] = {"Disabled","Enabled with Upgrade","Enabled"};
		default = 2;
	};
	class CTI_VEHICLES_AIR_AA {
		title = "UNITS: Aircraft AA Missiles";
		values[] = {0,1,2};
		texts[] = {"Disabled","Enabled with Upgrade","Enabled"};
		default = 2;
	};
	class CTI_VEHICLES_AIR_AT {
		title = "UNITS: Aircraft AT Missiles";
		values[] = {0,1,2};
		texts[] = {"Disabled","Enabled with Upgrade","Enabled"};
		default = 2;
	};
	class CTI_VEHICLES_AIR_CM {
		title = "UNITS: Aircraft Countermeasures";
		values[] = {0,1,2};
		texts[] = {"Disabled","Enabled with Upgrade","Enabled"};
		default = 2;
	};
	class CTI_MARKERS_INFANTRY {
		title = "UNITS: Show Map Infantry";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 1;
	};
	class CTI_UNITS_FATIGUE {
		title = "UNITS: Fatigue";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 0;
	};
	class CTI_UNITS_FUEL_CONSUMPTION {
		title = "UNITS: Increased Fuel Consumption";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 0;
	};
	class CTI_VEHICLES_EMPTY_TIMEOUT {
		title = "UNITS: Vehicles Reycling Delay";
		values[] = {60,120,180,240,300,600,1200,1800,2400,3000,3600};
		texts[] = {"1 Minute","2 Minutes","3 Minutes","4 Minutes","5 Minutes","10 Minutes","20 Minutes","30 Minutes","40 Minutes","50 Minutes","1 Hour"};
		default = 600;
	};
	class CTI_ARTILLERY_SETUP {
		title = "ARTILLERY: Setup";
		values[] = {-2,-1,0,1,2,3};
		texts[] = {"Disabled","Ballistic Computer","Short","Medium","Long","Extreme"};
		default = 1;
	};
	class SEPARATOR6 {
		title = "========================== GAMEPLAY ===========================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_GAMEPLAY_3P {
		title = "GAMEPLAY: 3P view";
		values[] = {-1,0,1,2};
		texts[] = {"All","No group view","Vehicle","None"};
		default = -1;
	};
	class CTI_WEAPON_SWAY {
		title = "GAMEPLAY: Weapon Sway Level";
		values[] = {0,25,50,75,100};
		texts[] = {"None","Little","Normal","Allot","Full"};
		default = 0;
	};
	class CTI_SM_NONV {
		title = "GAMEPLAY: Disable NVs, Thermal on Players";
		values[] = {0,1,2,3};
		texts[] = {"Default","Disable NV", "Disable Thermals","Disable NV/THERMALS"};
		default = 0;
	};
	class CTI_SM_NV_THER_VEH {
		title = "GAMEPLAY: Disable NVs, Thermal on Vehicles, Statics";
		values[] = {0,1,2,3};
		texts[] = {"Default","Disable NV", "Disable Thermals","Disable NV/THERMALS"};
		default = 0;
	};
	class CTI_GAMEPLAY_DARTER {
		title = "GAMEPLAY: Darter Range limitation";
		values[] = {0,500,1000,2000,4000,6000,8000};
		texts[] = {"None","Connection range 500m","Connection range 1000m","Connection range 2000m","Connection range 4000m","Connection range 6000m", "Connection range 8000m"};
		default = 0;
	};
	class SEPARATOR7 {
		title = "========================== TEAMPLAY ===========================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_TEAMSWAP {
		title = "TEAM: Team swap protection";
		values[] = {0,1};
		texts[] = {"off","on"};
		default = 1;
	};
	class CTI_TEAMSTACK {
		title = "TEAM: Team Stack protection";
		values[] = {0,1,2,3,4,5};
		texts[] = {"Disabled","+1 Player","+2 Players","+3 Players","+4 Players","+5 Players"};
		default = 2;
	};
	class SEPARATOR8 {
		title = "======================== MAP SETTINGS =========================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_GRAPHICS_TG_MAX {
		title = "VISUAL: Terrain Grid";
		values[] = {10,20,30,40,50};
		texts[] = {"Far","Medium","Short","Shorter","Free"};
		default = 50;
	};
	class CTI_GRAPHICS_VD_MAX {
		title = "VISUAL: View Distance";
		values[] = {1000,1500,2000,2500,3000,3500,4000};
		texts[] = {"1 KM","1.5 KM","2 KM","2.5 KM","3 KM","3.5 KM","4 KM"};
		default = 4000;
	};
	class CTI_WEATHER_INITIAL {
		title = "WEATHER: Inital Time";
		values[] = {-6,-5,-4,-3,-2,-1,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18};
		texts[] = {"12 AM","1 AM","2 AM","3 AM","4 AM","5 AM","6 AM","7 AM","8 AM","9 AM","10 AM","11 AM","12 PM","1 PM","2 PM","3 PM","4 PM","5 PM","6 PM","7 PM","8 PM","9 PM","10 PM","11 PM","Random"};
		default = -1;
	};
	class CTI_WEATHER_RAIN {
		title = "WEATHER: Rain (Rain Requires Overcast Greater High)";
		values[] = {-1,0,30,50,75,100};
		texts[] = {"Random","Clear","Light","Medium","High","Max"};
		default = 0;
	};
	class CTI_WEATHER_RAIN_COEF {
		title = "WEATHER: Rain Variance";
		values[] = {-1,0,0.1,0.25,0.5,0.75,1};
		texts[] = {"Random","None","10%","25%","50%","75%","Chaos"};
		default = 0;
	};
	class CTI_WEATHER_SNOW {
		title = "WEATHER: Snow Storms";
		values[] = {0,1,2,3,4};
		texts[] = {"Clear","Light","Medium","High","Max"};
		default = 0;
	};
	class CTI_WEATHER_SNOW_COEF {
		title = "WEATHER: Snow Variance";
		values[] = {-1,0,0.1,0.25,0.5,0.75,1};
		texts[] = {"Random","None","10%","25%","50%","75%","Chaos"};
		default = 0;
	};
	class CTI_WEATHER_DUST {
		title = "WEATHER: Dust Storms";
		values[] = {0,1,2,3,4};
		texts[] = {"Clear","Light","Medium - Adds Wall of Dust","High - Adds Effect on Objects","Max - Adds Lethal Wall"};
		default = 0;
	};
	class CTI_WEATHER_DUST_COEF {
		title = "WEATHER: Dust Variance";
		values[] = {-1,0,0.1,0.25,0.5,0.75,1};
		texts[] = {"Random","None","10%","25%","50%","75%","Chaos"};
		default = 0;
	};
	class CTI_WEATHER_MONSOON {
		title = "WEATHER: Monsoon Storms";
		values[] = {0,1,2};
		texts[] = {"Clear","No Damage","Damage Objects"};
		default = 0;
	};
	class CTI_WEATHER_MONSOON_COEF {
		title = "WEATHER: Monsoon Variance";
		values[] = {-1,0,0.1,0.25,0.5,0.75,1};
		texts[] = {"Random","None","10%","25%","50%","75%","Chaos"};
		default = 0;
	};
	class CTI_WEATHER_OVERCAST {
		title = "WEATHER: Overcast";
		values[] = {-1,0,30,50,75,100};
		texts[] = {"Random","Clear","Light","Medium","High","Max"};
		default = 0;
	};
	class CTI_WEATHER_OVERCAST_COEF {
		title = "WEATHER: Overcast Variance";
		values[] = {-1,0,0.1,0.25,0.5,0.75,1};
		texts[] = {"Random","None","10%","25%","50%","75%","Chaos"};
		default = 0;
	};
	class CTI_WEATHER_FOG {
		title = "WEATHER: Fog value";
		values[] = {-1,0,10,20,30,40,50,60,70,80,90,100};
		texts[] = {"Random","Clear","10","20","30","40","50","60","70","80","90","MAX"};
		default = 0;
	};
	class CTI_WEATHER_FOG_COEF {
		title = "WEATHER: Fog Variance";
		values[] = {-1,0,0.1,0.25,0.5,0.75,1};
		texts[] = {"Random","None","10%","25%","50%","75%","Chaos"};
		default = 0;
	};
	class CTI_WEATHER_FOG_DECAY {
		title = "WEATHER: Fog decay level";
		values[] = {-1,0,0.02,0.04,0.06,0.08,0.1,0.15,0.20,0.25,0.5,0.75,1};
		texts[] = {"Random","None","0.02","0.04","0.06","0.08","0.1","0.15","0.20","0.25","0.5","0.75","1"};
		default = 0;
	};
	class CTI_WEATHER_FOG_DECAY_COEF {
		title = "WEATHER: Fog decay Variance";
		values[] = {-1,0,0.1,0.25,0.5,0.75,1};
		texts[] = {"Random","None","10%","25%","50%","75%","Chaos"};
		default = 0;
	};
	class CTI_WEATHER_FOG_ALT {
		title = "WEATHER: Fog altitude";
		values[] = {-1,0,1,5,25,50,100,200,300,400,500,600,700,800,900,1000};
		texts[] = {"Random","0m","1m","5m","25m","50m","100m","200m","300m","400m","500m","600m","700m","800m","900m","1km"};
		default = -1;
	};
	class CTI_WEATHER_FOG_ALT_COEF {
		title = "WEATHER: Fog altitude Variance";
		values[] = {-1,0,0.1,0.25,0.5,0.75,1};
		texts[] = {"Random","None","10%","25%","50%","75%","Chaos"};
		default = 0;
	};
	class CTI_WEATHER_WIND {
		title = "WEATHER: Wind";
		values[] = {-1,0,30,50,75,100};
		texts[] = {"Random","Clear","Light","Medium","High","Max"};
		default = 0;
	};
	class CTI_WEATHER_WIND_COEF {
		title = "WEATHER: Wind Variance";
		values[] = {-1,0,0.1,0.25,0.5,0.75,1};
		texts[] = {"Random","None","10%","25%","50%","75%","Chaos"};
		default = 0;
	};
	class CTI_WEATHER_WAVES {
		title = "WEATHER: Waves";
		values[] = {-1,0,30,50,75,100};
		texts[] = {"Random","Clear","Light","Medium","High","Max"};
		default = 0;
	};
	class CTI_WEATHER_WAVES_COEF {
		title = "WEATHER: Waves Variance";
		values[] = {-1,0,0.1,0.25,0.5,0.75,1};
		texts[] = {"Random","None","10%","25%","50%","75%","Chaos"};
		default = 0;
	};
	class CTI_WEATHER_VARIANCE_TIME {
		title = "WEATHER: Frequency of weather changes (or between of storms)";
		values[] = {-1,0,60,120,300,600,900,1800,2700,3600,5400,7200};
		texts[] = {"Random","No Change","1min","2min","5min","10min","15min","30min","45min","60min","90min","120min"};
		default = 0;
	};
	class CTI_WEATHER_STORM_TIME {
		title = "WEATHER: Duration of Storms (dust storms and monsoons)";
		values[] = {-1,60,120,300,600,900,1800,2700,3600,5400,7200,14400};
		texts[] = {"Random","1min","2min","5min","10min","15min","30min","45min","60min","90min","120min","240min"};
		default = 900;
	};
  	class CTI_WEATHER_FAST {
		title = "WEATHER: Day Duration";
		values[] = {1,2,3,4,5,6,7,8,9,10,11,12};
		texts[] = {"1 hour","2 hours","3 hours","4 hours","5 hours","6 hours","7 hours","8 hours","9 hours","10 hours","11 hours","12 hours"};
		default = 12;
	};
  	class CTI_WEATHER_FAST_NIGHT {
		title = "WEATHER: Night Duration";
		values[] = {1,2,3,4,5,6,7,8,9,10,11,12};
		texts[] = {"1 hour","2 hours","3 hours","4 hours","5 hours","6 hours","7 hours","8 hours","9 hours","10 hours","11 hours","12 hours"};
		default = 12;
	};
	class SEPARATOR9 {
		title = "=========================== ADDONS ============================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_APEX_ADDON {
		title = "ADDON: APEX DLC Support";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 1;
	};
	class CTI_CUP_ADDON {
		title = "ADDON: CUP Support, Units, Vehicles, Weapons";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 0;
	};
	class CTI_CUP_CORE_ADDON {
		title = "ADDON: CUP CORE and CUP Terrains";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 1;
	};
	class CTI_OFPS_ADDON {
		title = "ADDON: OFPS MOD Pack Support";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 0;
	};
	class CTI_RHS_ADDON {
		title = "ADDON: RHS AFRF, RHS GREF, RHS SAF, RHS USAF Support";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 1;
	};
	class SEPARATOR10 {
		title = "========================= GAMEMODES ===========================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_ZOMBIE_MODE {
		title = "GAMEMODE: Zombie Mode";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 0;
	};
	class CTI_GUERILLA_MODE {
		title = "GAMEMODE: Guerilla Mode (Low Tech)";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 0;
	};
	class SEPARATOR11 {
		title = "========================== DEV MODE ===========================";
		values[] = {1};
		texts[] = {""};
		default = 1;
	};
	class CTI_DEV_MODE {
		title = "DEV: Dev Mode";
		values[] = {0,1};
		texts[] = {"Disabled","Enabled"};
		default = 0;
	};
};